src/GlobalErrorHandler.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+import { SafeLink, Text, colors } from '@aragon/ui'
+
+class GlobalErrorHandler extends React.Component {
+  state = { error: null }
+  componentDidCatch(error, info) {
+    this.setState({ error })
```

Comment:
Should we log the `info` here (not sure if React will still log the error if we catch it like this)? I'm curious as to what information gets provided here too.

Reply:
Updated, the components stack is now displayed in the screen itself.